### PR TITLE
UI refresh

### PR DESCRIPTION
**Pull request by techenby**
- [x] Make light mode more modern
- [x] Add dark mode
- [x] Bring tailwind variables up to v3 standard

Homepage - Light
Homepage -  Dark

Project Page - Light
Project Page - Dark

PR - Light
PR - Dark

Issue - Light
Issue - Dark

### CODE REVIEW

**Comment by jonsugar** [comment]
This is shaping up to be an awesome PR so far.

**Comment by mattstauffer** [comment]
These are some really great improvements, @techenby --thank you!

I have a few notes that are totally up for conversation.

1. I think the home/breadcrumbs tab is very cool looking! But from a perspective of how it fits in the whole site, it's actually distracting to me--because of where it is in the layout, it's the first place my eye goes. But this begs the question--why did we even add it in the first place? That's not your fault at all, it was there already and you just styled it beautifully lol... in the end, I think it doesn't provide any actual value (we never have more than one layer of breadcrumbs; the text in the breadcrumb is effectively the same as the page title; and the home button already can be achieved by clicking the logo). So.... sorry to waste your work! but I think we should drop it, unless anyone wants to make a srong case for it.

image

2. The overlapping "4 pull requests" thing looks so slick! could you put a bit more margin below it so it doesn't butt so close up to the top of the first pull request?

image

3. The visual weight of the metadata on the project page is a bit too heavy now that the text is darker. If you made it darker to meet accessibility guidelines, could we possibly make it smaller? And if not, could we lighten it a bit (or make it smaller, if you prefer?)

image

Again--really, really good work!

File: public/js/resources_ts_Pages_Issue_Show_vue.js [hunk 1]
```diff
@@ -0,0 +1,482 @@
+"use strict";
```
Comment (mattstauffer) [inline public/js/resources_ts_Pages_Issue_Show_vue.js:1]: Not a big deal, but do you know why these files showed up?

Comment (techenby) [reply public/js/resources_ts_Pages_Issue_Show_vue.js:1]: Nope, but I can look into it!

**Comment by techenby** [comment]
I agree that the breadcrumbs are not super useful or really fit into the structure of the page, but I didn't want to just blindly get rid of stuff so I tried to make it at least more cohesive.

**Comment by mattstauffer** [comment]
@techenby  :) Yep. You did a good thing, but because I'm the boss (and I made Ozzie originally) I just get to show up and delete shit lol... so, again, great work! But let's drop em.

**Comment by techenby** [comment]
For the meta data, I do think we can go lighter and/or smaller in light mode. But in dark mode they're as dark as can be accessibly wise.

**Comment by jonsugar** [comment]
> 1. I think the home/breadcrumbs tab is very cool looking! But from a perspective of how it fits in the whole site, it's actually distracting to me

Haha, I almost exclusively use the breadcrumbs for both navigations to the project page and home... I guess it's just a personal thing. I do find it useful as a constant reminder of my location but I do take the point that they are not super beneficial in that respect since the site only has one layer.

> The overlapping "4 pull requests" thing looks so slick! could you put a bit more margin below it so it doesn't butt so close up to the top of the first pull request?

Not my cup of tea but also just a stylistic thing. 

> @techenby :) Yep. You did a good thing, but because I'm the boss...

And, yes! you are. Just wanted to speak about the breadcrumbs mostly.

**Pull request closed without merging**